.github/workflows/e2e-external.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: End to End Tests for external scenario
+
+on:
+  pull_request:
+
+jobs:
+  e2e-external:
+    name: E2E-external
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2
+
+      - shell: bash
+        run: |
+          echo "::group::Reclaiming free space"
+          # Clean up tools we don't need for our CI to free up more space on the hosted runner
+          rm -rf /opt/ghc /usr/share/dotnet /usr/share/swift
+          df -h
+          echo "::endgroup::"
```

Comment:
I vaguely remember that we removed similar code from individual repos and wanted to use shared actions from Shipyard. @mangelajo @tpantelis might be knowing more on this.

Reply:
Maybe https://github.com/submariner-io/shipyard/issues/135 ?

Reply:
I think it's good if we just use the action here: 
https://github.com/submariner-io/shipyard/blob/devel/gh-actions/e2e/action.yaml#L14

But first we need to add a parameter for the make target, which would default to "e2e"

Reply:
Done